addon/services/media.js

```diff
@@ -183,27 +203,30 @@ export default Service.extend(Evented, {
   * @method  match
   */
   match(name, query) {
-    if (this.get('_mocked')) {
+    // see https://github.com/ember-cli/eslint-plugin-ember/pull/272
+    if (Ember.testing && this._mocked) {
       return;
     }
 
-    const matcher = this.get('mql')(query);
+    const mql = this.mql;
+    const matcher = mql(query);
 
     const listener = (matcher) => {
-      if (this.get('isDestroyed')) {
+      if (this.isDestroyed) {
         return;
       }
 
-      this.set(`matchers.${name}`, matcher);
+      set(this, `matchers.${name}`, matcher);
 
       if (matcher.matches) {
-        this.get('matches').addObject(name);
+        this.matches = Array.from(new Set([...this.matches, name]));
```

Comment:
Is this change to trigger native getters on the matches array change?

Reply:
Yep!  reset the value instead of pushing into it.  Otherwise we need tracked-built-ins.  Since this data structure is very low footprint, this seems reasonable.